src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -21,6 +21,7 @@
     <ImportVSSDKTargets>true</ImportVSSDKTargets>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <CopyNuGetImplementations>true</CopyNuGetImplementations>
+    <DeployExtension Condition="'$(VisualStudioVersion)' == '14.0'">false</DeployExtension>
```

Comment:
We define it in our core targets if it's unset, because we do rely on it everywhere.